db/schema.rb

```diff
@@ -10,10 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_07_18_203652) do
+ActiveRecord::Schema[7.1].define(version: 2024_05_27_184532) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "guests", force: :cascade do |t|
+    t.string "name"
+    t.integer "nights"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
Great job creating the guests table, the only issue with your database setup is that you are missing the join table between guests and rooms. In the README it says that Guests have many Rooms and Rooms have many Guests. And anytime you have a many-to-many relationship like this, it's a join and requires a join table.